homeassistant/components/conversation/__init__.py

```diff
@@ -39,6 +39,21 @@
     extra=vol.ALLOW_EXTRA,
 )
 
+WS_TYPE_GET_ONBOARDING = "conversation/onboarding/get"
+WS_TYPE_SET_ONBOARDING = "conversation/onboarding/set"
+WS_TYPE_GET_ATTRIBUTION = "conversation/attribution"
+
+SCHEMA_GET_ONBOARDING = websocket_api.BASE_COMMAND_MESSAGE_SCHEMA.extend(
```

Comment:
This is not how we define websocket commands anymore. Do this instead:

https://github.com/home-assistant/home-assistant/blob/fadb6a39790c688501d8415d94e0047aca40057b/homeassistant/components/config/config_entries.py#L249-L253